js/components/accessibilityControls.js

```diff
@@ -131,6 +131,10 @@ AccessibilityControls.prototype = {
    * @returns {boolean} true if moved
    */
   moveVrToDirection: function(e, charCode, isKeyDown, targetTagName) {
+    var keyDirectionMap = this.keyDirectionMap;
+    if (!(keyDirectionMap[charCode] || targetTagName !== "button")) {
```

Comment:
null check on keyDirectionMap. What if it is null, what if it is an empty array.